sources/europe/ge/ge_orthophoto_tbilisi_2014.geojson

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "GE_ORTOFOTO_TBILISI_2014",
+        "name": "Orthophoto (2014), 1:200, Tbilisi, Georgia",
+        "type": "tms",
+        "url": "http://mapstiles.tbilisi.gov.ge/tbilisimap-tileserver/tilesproxy?x={x}&y={y}&z={z}&l=msda-satellite",
+        "country_code": "GE",
+        "available_projections": [
+            "EPSG:32638",
+            "EPSG:4326",
+            "EPSG:3857",
+            "EPSG:900913"
+        ],
+        "attribution": {
+            "text": "The owner of the raster data is the Tbilisi City Hall, the copy is kept in the State Fund of Geodesy and Cartography of the National Agency of Public Registry."
+        },
+        "license_url": "http://mp1.napr.gov.ge/ORTHO_2014_TBILISI/service?REQUEST=GetCapabilities",
```

Comment:
Although this lists AccessConstraints as `none`, I'm not familial with Georgian copyright law and government copyright, is there any documentation I can read on the legality of using this imagery for tracing in OSM?

Reply:
I contacted the official that is responsible for these maps and he allowed OSM community to use this orthophoto maps for mapping/tracing. But unfortunately I couldn't get him to write this on the municipal website. I think he doesn't really understand, what more I need to proceed. Now I am stuck.

Reply:
You can find a template text for aerial imagery tracing waiver at https://wiki.osmfoundation.org/wiki/Licence/Waiver_and_Permission_Templates could they complete that?

Reply:
I found the [official webpage](http://geonetwork.napr.gov.ge/geonetwork/srv/api/records/89b93c6b-dc1e-4e58-b5aa-81003d733464) of the map with details and license. I have asked the official to edit the license. Let's wait.

Reply:
Thanks.

"Use for commercial purposes is prohibited" so definitely not suitable for use in OSM as it stands.